package.json

```diff
@@ -77,6 +77,7 @@
     "postcss-cli": "^10.0.0",
     "postcss-import": "^14.1.0",
     "postcss-mixins": "^9.0.3",
+    "postcss-nested": "^6.0.0",
```

Comment:
So far we haven been adding a `& ` anywhere a nested class is used:

```diff
  .some-class {
-   .octicon {
+   & .octicon {
      ...
    }
  }
```

but I assume `postcss-nested` would allow us to keep using the "SCSS nesting" without the `& `. @jonrohan also mentioned that it might will become a CSS standard. So using `postcss-nested` might be fine?

Reply:
We're trying to write closer to CSS spec. That said they are talking about changing the specification for nesting and there's an experimental plugin we could use to try it. https://twitter.com/css_tools_/status/1587392707206561792?s=46&t=OsB8hQIe4u2xhsK3WD3iYw

https://twitter.com/css_tools_/status/1587392697857413120?s=46&t=OsB8hQIe4u2xhsK3WD3iYw